samples/CoreAnimationTest/CoreAnimationTest/HeaderTableViewCell.m

```diff
@@ -59,15 +56,33 @@ - (instancetype)initWithReuseIdentifier:(NSString*)reuseIdentifier {
                                                                      options:0
                                                                      metrics:metrics
                                                                        views:views]];
+
+        // Separator
+        _separator = [CALayer layer];
+        _separator.backgroundColor = [UIColor darkGrayColor].CGColor;
+        _separator.frame = CGRectMake(0, CGRectGetHeight(self.frame) - 0.5f, CGRectGetWidth(self.frame), 0.5f);
+        [self.layer addSublayer:_separator];
     }
 
     return self;
 }
 
+- (void)layoutSubviews {
+    [super layoutSubviews];
+    _separator.frame = CGRectMake(0, CGRectGetHeight(self.frame) - 0.5f, CGRectGetWidth(self.frame), 0.5f);
+}
+
 - (void)setUpWithView:(UIView*)view andTitle:(NSString*)title {
     _titleLabel.text = title;
     _countLabel.text = [NSString stringWithFormat:@"%lu layers", view.layer.sublayers.count + 1];
+    _countLabel.hidden = NO;
     _layerView = view;
 }
 
+- (void)setUpWithTitle:(NSString*)title {
+    _titleLabel.text = title;
+    _countLabel.hidden = YES;
+    _layerView = nil;
+}
```

Comment:
It could but I wanted the separation in case I expanded the cell and added more logic. I also wanted to avoid passing nil to a view in the combined version.,

---
In reply to: [99437407](https://github.com/Microsoft/WinObjC/pull/1874#discussion_r99437407) [](ancestors = 99437407)